File: nodejs/7 kyu/168.js
// https://www.codewars.com/kata/52aeb2f3ad0e952f560005d3/train/javascript

// Passed

function sortGiftCode(code) {
  let sortedLetters = code.split("").sort();
  let sortedGiftCode = sortedLetters.join("");
  return sortedGiftCode;
}

const output = sortGiftCode("pqksuvy");
console.log(output);

File: python/7 kyu/168.py
# https://www.codewars.com/kata/59a1cdde9f922b83ee00003b/train/python

# Passed

def stanton_measure(arr):
    times_one_appears = arr.count(1)
    stanton_measure_value = arr.count(times_one_appears)
    
    return stanton_measure_value

output = stanton_measure([1, 4, 3, 2, 1, 2, 3, 2])
print(output)